CONTRIBUTING.md

```diff
@@ -191,3 +191,41 @@ these steps:
 >    ```
 >    The error is related to the `tcl` package. You should uninstall the `tcl`
 > package to resolve the error.
+
+## Error handler
+
+Use the `DPCTL_VERBOSITY` environment variable to show errors of different severity levels of the C-API in the console.
```

Comment:
s/to show errors of different severity levels of the C-API in the console/to print to the console errors of different severity levels from the libsyclinterface library/g